### PR TITLE
Fix/spacing in footer section

### PR DESCRIPTION
**Pull request by Fidesnoella**
# Description

Add  margin left to the footer and font size to tablet break point.

Fixes # (issue)

Fix font size to tablet version

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# How Has This Been Tested?

- [x] npm run dev
- [x] npm run build

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -62,6 +62,9 @@ module.exports = {
       width: {
         78: "19.375rem",
       },
+      maxWidth: {
```
Comment (esther921) [inline tailwind.config.js:95]: Don't you think that with changing the maxWidth you will change alot on the wrapper set I suggest you keep this and also add the maxwidth for the wrapper!

File: tailwind.config.js [hunk 2]
```diff
@@ -62,6 +62,9 @@ module.exports = {
...
+        "6.5xl": "76.3125rem"
```
Comment (esther921) [inline tailwind.config.js]: ` maxWidth: {
        90: "90rem",
      },`

This is the standard thin and you can try to add it to and test the effect on your wrapper

**Review by esther921 — CHANGES_REQUESTED** [review]

**Review by Stevenkwizera06 — CHANGES_REQUESTED** [review]
Good work Noella try to change in the Impressum Privacy add a slash between those two words

**Review by esther921 — APPROVED** [review]
LGTM

**Review by NNesta — APPROVED** [review]
LGTM

**Review by Stevenkwizera06 — APPROVED** [review]

**Pull request merged**